test/e2e/lib/gutenberg/blocks/gallery-masonry-block-component.js

```diff
@@ -12,6 +12,23 @@ class GalleryMasonryBlockComponent extends GutenbergBlockComponent {
 	static blockTitle = 'Masonry';
 	static blockName = 'coblocks/gallery-masonry';
 	static blockFrontendSelector = By.css( '.entry-content .wp-block-coblocks-gallery-masonry' );
+
+	/**
+	 * Uploads images to the gallery.
+	 *
+	 * NOTE/TODO This is common to a few gallery-like blocks. DRY it using a mixin or common base class.
+	 *
+	 * @param {{imageName: string, fileName: string, file: string}} filesDetails a list of fileDetails
+	 */
+	async uploadImages( filesDetails ) {
+		const fileInputSelector = By.css( `div[id="${ this.blockID.slice( 1 ) }"] input[type=file]` );
+
+		const fileInput = this.driver.findElement( fileInputSelector );
+
+		const files = filesDetails.map( ( f ) => f.file ).join( '\n ' );
+
+		fileInput.sendKeys( files );
```

Comment:
Can we use the `setWhenSettable` helper?

Reply:
Sure, I honestly didn't think about it when I first wrote this 👍🏻 let's do it in a separate PR (maybe grouping several of these improvements together?)